src/extension.ts

```diff
@@ -438,7 +438,7 @@ export function activate(context: vscode.ExtensionContext) {
 
             const options = <vscode.QuickPickOptions> {
                 matchOnDescription: vscode.workspace.getConfiguration("projectManager").get("filterOnFullPath", false),
-                matchOnDetail: false,
+                matchOnDetail: true,
```

Comment:
The `detail` piece does not contain the project path. It only contains a warning about `paths not found`, displayed below the project name/path.

The project path is stored in the `description` property. So, I think the `filterOnFullPath` setting already does what you want.